haris2055/SOEN390
# Job Post Features

Note: If you are testing, upon checking out to this branch, you will need to run `python manage.py migrate` - I did not commit the DB file to avoid conflicts when merging to main.

This pull request includes the implementation of the following user stories: 

- #116 
To test this feature:
  - go to login page
  - sign in to a valid account
  - go to the 'Jobs' page from the Header
  - click 'Create a job'
  - fill out the form
  - press the Save button
  - **Expected Result**: The newly created job post should now be visible on the jobs page.

- #117
To test this feature:
  - go to login page
  - sign in to a valid account
  - go to the 'Jobs' page from the Header
  - either create a new job or edit an existing job
  - from the list, select which documents are required to apply for the job
  - press the Save button
  - **Expected Result**: The job post should now display a "Required Documents" section.

- #118 
To test this feature:
  - go to login page
  - sign in to a valid account
  - go to the 'Jobs' page from the Header
  - after having created at least one Job post, click the edit button on the Job Post
  - modify the form fields as needed
  - press the Save button
  - **Expected Result**: The job post should now display the updated information.

- #134 
To test this feature: 
  - go login page
  - sign in to a valid account
  - click on the 'Jobs' link in the header

If merged, this pull request closes #116, closes #117, closes #119, and closes #134.
